py/copygenerateducd.py

```diff
@@ -17,9 +17,10 @@
 
 
 def main():
+    out_of_source = '--out-of-source' in sys.argv[1:]
```

Comment:
As a reminder for our future selves: if we need to use additional args for this script, we should probably make use of Python's `argparse` module for robust handling. Not a concern for this PR though.